code/__DEFINES/sight.dm

```diff
@@ -57,3 +59,19 @@
 /// Bitfield of sight flags that show THINGS but no lighting
 /// Since lighting is an underlay on turfs, this is everything but that
 #define SEE_AVOID_TURF_BLACKNESS (SEE_MOBS|SEE_OBJS)
+
+//------------------------
+// INVISIBILITY PRIORITIES
+
+#define INVISIBILITY_PRIORITY_ADMIN 	100
+#define INVISIBILITY_PRIORITY_BASIC_ANTI_INVISIBILITY 1
+#define INVISIBILITY_PRIORITY_NONE		0
```

Comment:
```suggestion
#define INVISIBILITY_PRIORITY_ADMIN 100
#define INVISIBILITY_PRIORITY_BASIC_ANTI_INVISIBILITY 1
#define INVISIBILITY_PRIORITY_NONE 0
```

no idea what was going on with the indentation here

Reply:
When define numbers are important relative to one another I prefer when the values are lined up. Though I forgot to do it to anti invisibility as that was added around the end of all this.